include/ginkgo/core/matrix/csr.hpp

```diff
@@ -162,25 +222,77 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
         int64_t clac_size(const int64_t nnz) { return 0; }
     };
 
+    /**
+     * sparselib is a strategy_type which uses the sparselib csr.
+     */
+    class sparselib : public strategy_type {
+    public:
+        /**
+         * Creates a sparselib strategy.
```

Comment:
You can add a note here that says that you use `cuSparse` for CUDA and `hipSparse` for HIP.